.travis.yml

```diff
@@ -1,28 +1,25 @@
 sudo: false
 language: node_js
-cache:
-  directories:
-    - node_modules
 notifications:
   email: false
 node_js:
   - 4
   - 6
 matrix:
-    fast_finish: true
-    allow_failures:
+  fast_finish: true
+  allow_failures: ~
```

Comment:
This is one of the ways YAML node modules will output null values. They can also do `null` or `NULL` but I couldn't get it to leave them blank like we have it now.